core/src/test/java/ai/timefold/solver/core/impl/domain/solution/descriptor/SolutionDescriptorTest.java

```diff
@@ -157,12 +157,13 @@ void noProblemFactPropertyWithEasyScoreCalculation() {
 
     @Test
     void extended() {
-        var solutionDescriptor = TestdataAnnotatedExtendedSolution.buildExtendedSolutionDescriptor();
+        var solutionDescriptor = TestdataBothAnnotatedExtendedSolution.buildSolutionDescriptor();
         assertThat(solutionDescriptor.getProblemFactMemberAccessorMap()).isEmpty();
         assertThat(solutionDescriptor.getProblemFactCollectionMemberAccessorMap()).containsOnlyKeys("valueList",
                 "subValueList");
-        assertThat(solutionDescriptor.getEntityMemberAccessorMap()).isEmpty();
-        assertThat(solutionDescriptor.getEntityCollectionMemberAccessorMap()).containsOnlyKeys("entityList", "subEntityList");
+        assertThat(solutionDescriptor.getEntityMemberAccessorMap()).containsOnlyKeys("entity", "subEntity");
```

Comment:
This changes the logic of the test to the exact opposite!
This tells me that the two domains were not actually duplicates.

The test does a different thing now. That should not be allowed.

Reply:
I understand your point. However, I had to modify the existing inheritance classes to align with the deleted classes. This is why two new fields were added. I know the model change affects the tests, but the main goal of the test remains the same.